<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Добавить автомобиль</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Отсортировать по</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Добавить владельца</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Отсортировать по</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Марка</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Марке</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Марке(-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Году выпуска</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Году выпуска(-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Объёму двигателя</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Объёму двигателя(-)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="2">
     <widget class="QComboBox" name="comboBox_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>ФИО</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ФИО(-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дате</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дате(-)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Категории</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Категории(-)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Год выпуска</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>Сохранить в файл .dat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Дата вод. прав</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>Сохранить  файл .dat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Объём двигателя</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>Загрузить из файла .dat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Категория прав</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string>Загрузить из файла .dat</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_12">
      <property name="text">
       <string>Сохранить в файл .txt</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_14">
      <property name="text">
       <string>Сохранить в файл .txt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_13">
      <property name="text">
       <string>Загрузить из файла .txt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_15">
      <property name="text">
       <string>Загрузить из файла .txt</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать связь</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Установить связь</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать связь</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" rowspan="2">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="4" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Связать</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6" rowspan="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="7" colspan="2">
     <widget class="QSpinBox" name="spinBox_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QPushButton" name="pushButton_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать все</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7" colspan="3">
     <widget class="QPushButton" name="pushButton_7">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Показать все</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Марка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Год выпуска</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Объём двигателя</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="9" column="5" colspan="5">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата вод. прав</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Категория прав</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>spinBox</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_5</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_6</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>spinBox_4</zorder>
   <zorder>label_7</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>tableWidget</zorder>
   <zorder>tableWidget_2</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>pushButton_15</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
